src/viewer/Viewer.jsx

```diff
@@ -23,19 +23,27 @@ const isCordova = () => window.cordova !== undefined
 
 export default class Viewer extends Component {
   componentDidMount() {
+    this.fetchMore()
     document.addEventListener('keyup', this.onKeyUp, false)
   }
 
   componentWillUnmount() {
     document.removeEventListener('keyup', this.onKeyUp, false)
   }
 
+  fetchMore() {
+    if (this.props.files.length - this.props.currentIndex <= 5) {
+      this.props.fetchMore()
+    }
+  }
```

Comment:
Besides [the previous comment](https://github.com/cozy/cozy-drive/pull/560/commits/897733450d7f5465e54777ea69639b7362342b79#r155543603), I decide to fetch more files within two conditions:
1. if you click the `onNext` button and there is less than 6 files
1. if you open the viewer on a file under than the 5 last files

The other way was to call fechMore in the `render` function, but as the render function is called twice when it mounts and when we call `onNext` (I don't know why)

![](https://i.imgur.com/vhYBxin.png)